src/plugins/content/accessibility/jumplinks/plugin.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+
+/**
+ * @author aliaksandr paliakou <lecosson@gmail.com>
+ */
+
+import $ from 'jquery';
+import pluginFactory from 'taoTests/runner/plugin';
+import jumplinksFactory from "./jumplinks";
+import shortcutsFactory from "./shortcuts";
+
+function findFocusable(targetElement) {
+    const $elem = $(targetElement)
```

Comment:
```js
return $(targetElement)
  .find('input, select, a[href], textarea, button, [tabindex]')
  .toArray()
  .filter((el) => (el.tabIndex >= 0 && !el.disabled && el.offsetParent ) )
  .find((el) => (typeof el.focus === 'function') );
```

Reply:
refactored to fix support keynavigation plugin